Assignments/Splitwise/Python/README.md

```diff
@@ -1,3 +1,32 @@
 # Detailed Approach - Explain the approach how Problem was solved
 
+ - `class user` stores the information of each user like `userId`, `name`, `age`(private) and `mobile`(private)
+
+
+In `main.py`  -
+
+ - `class ProcessInput` defines all the methods that are specified in the instruction set i.e. `ADD`, `EXPENSE` and `SHOW`
+ 
+ - `class ProcessInput` also has two attributes namely `users` and `owes`
+
+ - Attribute `users` is a dictionary that stores instances of `user` class as values and their respective `user.userId` as the corresponding key.
+
+ - Attribute `owes` stores the EXPENSE details as the amount owed as the value and the tuple of 2 users as the corresponding key. This helps in saving some memory and computation time compared to saving an instance of each user mapped to another user.
```

Comment:
You are correctly solving as per given requirements. However, if additional requirements arrive, will you rwrite the entire code. Let us say this code is worth of more than 5K to 10K lines, will you rewrite the entire lot ?

Can you come up with more efficient design

Reply:
I think the previous comment also addresses this issue.

Reply:
You should consider documenting it in your approach so that this will be regarded as your work